pulsar-common/src/main/java/org/apache/pulsar/common/api/Commands.java

```diff
@@ -189,9 +194,16 @@ public static ByteBuf newConnect(String authMethodName, AuthData authData, int p
         return res;
     }
 
-    public static ByteBuf newConnected(int clientProtocolVersion) {
+    public static ByteBuf newConnected(int clientProtocoVersion) {
+        return newConnected(clientProtocoVersion, INVALID_MAX_MESSAGE_SIZE);
```

Comment:
Why do we need the `INVALID_MAX_MESSAGE_SIZE`? Is this method called from unit tests? I'd rather just leave the method which requires the `maxMessageSize`

Reply:
Because it will be used when there no max message size in protocol, like this:
```
ChannelFuture channelFuture;
            if (connected.hasMaxMessageSize()) {
                channelFuture = inboundChannel.writeAndFlush(
                    Commands.newConnected(connected.getProtocolVersion(), connected.getMaxMessageSize()));
            } else {
                channelFuture = inboundChannel.writeAndFlush(Commands.newConnected(connected.getProtocolVersion()));
            }
```